app/build.gradle

```diff
@@ -156,6 +156,7 @@ android {
 
 androidExtensions {
     experimental = true
+    features = ["parcelize"]
```

Comment:
we are not using @Parcelize yet

Reply:
So should we disable parcelize? This disables synthetics, by not mentioning them.

Reply:
this was also my question. So if you are not listing features explicitly everything is enabled?